is_area_empty.lua

```diff
@@ -17,6 +17,17 @@ end)
 
 
 jumpdrive.is_area_empty = function(pos1, pos2)
+	
+	minetest.emerge_area({ -- generates or loads the area before checking emptiness
+	x= pos1.x > 0 and pos1.x+16 or pos1.x-16, -- expands area by 16 in each axis to prevent border overlap 
+	y= pos1.y > 0 and pos1.y+16 or pos1.y-16,
+	z= pos1.z > 0 and pos1.z+16 or pos1.z-16
+	}, {
+	x= pos2.x > 0 and pos2.x+16 or pos2.x-16,
+	y= pos2.y > 0 and pos2.y+16 or pos2.y-16,
+	z= pos2.z > 0 and pos2.z+16 or pos2.z-16
```

Comment:
This won't work, the comparison needs to be between the two positions, not with 0.
```suggestion
	x= pos1.x > pos2.x and pos1.x+16 or pos1.x-16, -- expands area by 16 in each axis to prevent border overlap 
	y= pos1.y > pos2.y and pos1.y+16 or pos1.y-16,
	z= pos1.z > pos2.z and pos1.z+16 or pos1.z-16
	}, {
	x= pos2.x > pos1.x and pos2.x+16 or pos2.x-16,
	y= pos2.y > pos1.y and pos2.y+16 or pos2.y-16,
	z= pos2.z > pos1.z and pos2.z+16 or pos2.z-16
```

Reply:
@PolySaken-I-Am: did you try this without any calculations? i believe the emerging process rounds up anyway